test/Core.Test/TempDirectory.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.IO;
+
+namespace Bit.Core.Test
+{
+    public class TempDirectory : IDisposable
```

Comment:
I got scared away from that library since it would require injecting a new FileSystem dependency and I wasn't too sure about its popularity.

I'm willing to give it a shot with more support than just from myself, though. I 100% agree with your breakdown of why we want it.

Reply:
I'm good with another approach, for now we can stick with this as-is, just stinks Xunit doesn't provide the facilities to do this out of the box. Let's not add anything like this at this time. thanks!

Reply:
Xunit seems to take the stance quite frequently that you should roll your own :(. Looking at you, Assertions, Before/After Each/All!